ddclient.in

```diff
@@ -983,6 +983,22 @@ my %services = (
             %{$variables{'service-common-defaults'}},
         },
     },
+    'route53' => {
+        'updateable' => undef,
+        'update'     => \&nic_route53_update,
+        'examples'   => \&nic_route53_examples,
+        'variables'  => {
+            'login' => setv(T_STRING, 0, 0, 'unused', undef),
```

Comment:
Consider adding a comment explaining why the values are what they are. I assume these are just nulled out, but I see in `hosted-zone-id` there's a `1`, and I'm confused why.

Reply:
Done, please let me know if these new comments are clear enough.

Reply:
Looks great!